src/sqs/backfill.ts

```diff
@@ -7,13 +7,10 @@ export type BackfillMessagePayload = {
 	jiraHost: string
 }
 
-export const backfillQueueMessageHandler : MessageHandler<BackfillMessagePayload> = {
-
-	async handle(context) {
-		if(!await booleanFlag(BooleanFlags.NEW_BACKFILL_PROCESS_ENABLED, false, context.payload.jiraHost)) {
-			context.log.info("New backfill process disabled, dropping the message ");
-			return
-		}
-		context.log.info("Porcessing backfill message");
+export const backfillQueueMessageHandler : MessageHandler<BackfillMessagePayload> = async (context) => {
```

Comment:
I had to address some PR comments and made this as a function instead of an object. No other changes in this file